### ISSUE TITLE
ACC-262.6, ACC-262.7 (Level AAA) _ Modify Modules _  _ Application settings gets overridden by the OS settings 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3138]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	-
Affected Check Point:	ACC-262.6,   ACC-262.7
Short text:	ACC-262.6, ACC-262.7 (Level AAA) _ Modify Modules _  _ Application settings gets overridden by the OS settings 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Activate ‘Profile’ menu button and select ‘Preferences’ menu item. Now select any theme from the listed themes e.g. ‘Light/Dark/High Contrast Black and White’ themes one at a time.
•	Now open ‘Settings’ from the Windows desktop  and select ‘Accessibility’ and select ‘Contrast themes’.
•	Now add  a new device using default ‘Aquatic’ theme for High contrast Black theme and editing the background colour as #000000 and save and do the same for High Contrast White but use default theme ‘Dessert’ and  select background colour as #FFFFFF and apply the settings one at a time..

3. Issue Type: Manual
Observed Behavior:
On applying any theme eg:  ‘Light/Dark/High Contrast Black and White’ themes in the application and changing the OS settings to ‘Light/Dark/High Contrast Black and White’ theme or any other theme, the theme in the application is overridden by the theme applied through OS settings  thus confusing the user.
Screenshot:
In desert theme:
 
Case 2: in high contrast black theme
 

case 3: when the same theme is applied in Application and OS settings
 
After:
 
Case 2:
 
After:
 

Expected Behavior:
The application theme shouldn’t be overridden by the theme in OS settings. Whatever theme is applied in the application should remain intact in spite of changing the OS settings.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-262.6, ACC-262.7 (Level AAA) _ Modify Modules _  _ Application settings gets overridden by the OS settings.docx](https://github.com/user-attachments/files/16659838/ACC-262.6.ACC-262.7.Level.AAA._.Modify.Modules._._.Application.settings.gets.overridden.by.the.OS.settings.docx)
[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659840/Issue.processing-09.2021.002.docx)

**Comment by mrCherry97** [comment]
Couldn't reproduce, covered by UI5.
